src/indicators/flooding/glofas/utils.py

```diff
@@ -250,13 +251,14 @@ def get_crps(
             observations = ds_reanalysis[station].reindex(
                 {"time": forecast.time}
             )
+
             if normalization == "mean":
                 norm = observations.mean().values
             elif normalization == "std":
                 norm = observations.std().values
             elif normalization is None:
                 norm = 1
```

Comment:
maybe just use `else` instead of checking for `None`, and then set `norm = normalization` (that way other values can be passed through)